--- openbox-menu-0.3.6.6/Makefile.orig	2011-11-13 13:34:04.000000000 +0000
+++ openbox-menu-0.3.6.6/Makefile	2012-04-26 14:13:47.822229041 +0000
@@ -1,6 +1,17 @@
 LIBS= `pkg-config --libs glib-2.0 gtk+-2.0 libmenu-cache`
 CFLAGS= -Wall -O -g `pkg-config --cflags glib-2.0 gtk+-2.0 libmenu-cache`
 
+DESTDIR=
+prefix=/usr
+BINDIR=$(prefix)/bin
+DESKTOPDIR=$(prefix)/share/applications
+DOCDIR=$(prefix)/share/doc/$(PKGNAME)-$(MKLIVECDVER)
+ICONSDIR=$(prefix)/share/pixmaps
+LIBDIR=$(prefix)/lib/$(PKGNAME)
+LOCALEDIR=$(prefix)/share/locale
+SBINDIR=$(prefix)/sbin
+SHAREDIR=$(prefix)/share/$(PKGNAME)
+
 SRC= $(shell ls *.c 2> /dev/null)
 OBJ= $(SRC:.c=.o)
 
@@ -17,3 +28,7 @@
 
 clean:
 	@rm *.o openbox-menu
+
+install:
+	@install -dm 755 $(DESTDIR)$(BINDIR)
+	@install -m 755 openbox-menu $(DESTDIR)$(BINDIR)
